<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportRevealDialog</class>
 <widget class="QDialog" name="ExportRevealDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to reveal.js</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="exportToLabel">
     <property name="text">
      <string>E&amp;xport to:</string>
     </property>
     <property name="buddy">
      <cstring>exportToLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="exportToLineEdit">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="choosePathButton">
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>&amp;Author:</string>
     </property>
     <property name="buddy">
      <cstring>authorLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="authorLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>&amp;Description:</string>
     </property>
     <property name="buddy">
      <cstring>descriptionTextEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="descriptionTextEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="showControlsCheckBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the controls to got back and forward in the right bottom corner&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Show controls at the bottom of the page</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="showProgressCheckBox">
     <property name="text">
      <string>Show &amp;progress</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="useHistoryCheckBox">
     <property name="text">
      <string>Use &amp;browser history</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="centerPageCheckBox">
     <property name="text">
      <string>C&amp;enter page on screen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportRevealDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportRevealDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>choosePathButton</sender>
   <signal>clicked()</signal>
   <receiver>ExportRevealDialog</receiver>
   <slot>choosePathButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportToLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ExportRevealDialog</receiver>
   <slot>exportToTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
